plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1409,27 +1442,47 @@ type VolumeNodeChecker struct {
 	client  clientset.Interface
 }
 
+// This predicate has been replaced by the VolumeBinding predicate
 // NewVolumeNodePredicate evaluates if a pod can fit due to the volumes it requests, given
 // that some volumes have node topology constraints, particularly when using Local PVs.
 // The requirement is that any pod that uses a PVC that is bound to a PV with topology constraints
 // must be scheduled to a node that satisfies the PV's topology labels.
-func NewVolumeNodePredicate(pvInfo PersistentVolumeInfo, pvcInfo PersistentVolumeClaimInfo, client clientset.Interface) algorithm.FitPredicate {
+func NewVolumeNodePredicate(pvInfo PersistentVolumeInfo, pvcInfo PersistentVolumeClaimInfo) algorithm.FitPredicate {
```

Comment:
I should just remove this.  This predicate is for an alpha feature so we don't support upgrades anyway.